packages/page-accounts/src/Accounts/modals/Delegate.tsx

```diff
@@ -31,10 +31,15 @@ function Delegate ({ amount: _amount, conviction: _conviction, delegatedAccount:
   const [delegatedAccount, setDelegatedAccount] = useState<string | null>(_delegatedAccount || null);
   const [conviction, setConviction] = useState(_conviction?.toNumber() || 1);
 
+  const isDirty = amount?.toString() !== _amount?.toString() ||
+    delegatedAccount !== _delegatedAccount ||
+    delegatingAccount !== _delegatingAccount ||
+    conviction !== _conviction?.toNumber();
+
   return (
     <Modal
       className='staking--Delegate'
-      header= {t<string>('Delegate democracy vote')}
+      header= {_delegatedAccount ? t<string>('democracy vote delegation') : t<string>('delegate democracy vote')}
```

Comment:
```suggestion
      header={_delegatedAccount ? t<string>('democracy vote delegation') : t<string>('delegate democracy vote')}
```

Generally would prefer the "?:" over multiple lines as well. (Not a crisis, just alignment)

Reply:
Sure, changed this.